schema.near.graphql

```diff
@@ -0,0 +1,25 @@
+type NearPayment @entity {
```

Comment:
could you add comments to know which fields are different from the other schema?